test/branded_token/accept_stake_request.js

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const BN = require('bn.js');
+const { AccountProvider } = require('../test_lib/utils.js');
+const { Event } = require('../test_lib/event_decoder.js');
+
+const web3 = require('../test_lib/web3.js');
+const utils = require('../test_lib/utils');
+const brandedTokenUtils = require('./utils');
+
+contract('TestBrandedToken::requestStake', async () => {
```

Comment:
Global comment (applies to constant/variable names, too, and to all test files): "TestBrandedToken" -> "BrandedToken"

That it's a test contract is immaterial for these purposes; additionally, when these tests point to the genuine BT contract, "TestBrandedToken" or "testBrandedToken" will need to be changed—might as well save the future developer the trouble/confusion.